inference-engine/thirdparty/clDNN/api/engine.hpp

```diff
@@ -86,6 +87,7 @@ struct engine_configuration {
         uint16_t n_streams = 1,
         const std::string& kernels_cache_path = "",
         uint16_t n_threads = std::max(static_cast<uint16_t>(std::thread::hardware_concurrency()), static_cast<uint16_t>(1)),
+        bool enable_loop_unrolling = true,
```

Comment:
where is this used? I think unrolling config is required only in the IE.

Reply:
remove it. thanks